src/javascript/app/common/form_manager.js

```diff
@@ -3,13 +3,23 @@ const BinarySocket     = require('../base/socket');
 const getHashValue     = require('../../_common/url').getHashValue;
 const isEmptyObject    = require('../../_common/utility').isEmptyObject;
 const showLoadingImage = require('../../_common/utility').showLoadingImage;
+const getElementById   = require('../../_common/common_functions').getElementById;
```

Comment:
should be on line 3

Reply:
No need for this line anymore, will proceed to remove it.